Claude-tag-best-practices.html

```
@@ -0,0 +1,1616 @@
+        font-size: 0.9rem;
+        margin: 0;
+      }
+      .step-card h4 {
+        margin: 0 0 0.4rem;
+        font-size: 0.98rem;
+        color: var(--text);
+      }
+    </style>
+  </head>
+  <body>
+    <button class="sidebar-toggle" id="navToggle" aria-label="目次を開く">☰ 目次</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)